encoders/pom.xml

```diff
@@ -104,6 +104,19 @@
       <artifactId>hapi-fhir-validation-resources-r4</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.skyscreamer</groupId>
+      <artifactId>jsonassert</artifactId>
+      <version>1.5.0</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>android-json</artifactId>
+          <groupId>com.vaadin.external.google</groupId>
+        </exclusion>
+      </exclusions>
+      <scope>test</scope>
+    </dependency>
```

Comment:
We already have this in `dependencyManagement` in the parent POM, so you should be able to reduce this to:

```xml
<dependency>
  <groupId>org.skyscreamer</groupId>
  <artifactId>jsonassert</artifactId>
</dependency>
```

Reply:
Ok. Actually this dependency was not fully defined in master pom, but it's fixed now.